src/pages/finished_screen.ui

```diff
@@ -12,8 +12,8 @@
           <object class="AdwStatusPage">
             <property name="hexpand">true</property>
             <property name="vexpand">true</property>
-            <property name="title">Installation finished!</property>
-            <property name="description">Reboot to finish the installation</property>
+            <property name="title">Installation Completed!</property>
+            <property name="description">Restart the device to start Crystal Linux!</property>
```

Comment:
I would change it to
"restart the device to boot crystal linux", as having 'start' twice sounds weird to me.

Reply:
What about "Reboot the device to start using Crystal Linux!"?

Reply:
IMO That's fantastic actually

Reply:
> 

yeah that's better